packages/pluggableWidgets/datagrid-web/src/components/Header.tsx

```diff
@@ -23,13 +23,15 @@ export interface HeaderProps<D extends object> {
     filterable: boolean;
     draggable: boolean;
     dragOver: string;
+    hidable: boolean;
     isDragging?: boolean;
-    visibleColumns: Array<ColumnInstance<D>>;
+    preview?: boolean;
```

Comment:
Where does this `preview` come from, I don't see a change in `Datagrid.xml`? What does this stand for? Can we make this name a bit more descriptive? 😄

Reply:
Preview is a prop being applied in .editorPreview file to indicate is Design mode / Studio